package.json

```diff
@@ -27,7 +27,7 @@
     }
   },
   "dependencies": {
-    "@gnosis.pm/dex-contracts": "^0.2.6",
+    "@gnosis.pm/dex-contracts": "^0.2.8",
```

Comment:
What happened to 7? Or is it just unlucky.

Reply:
It was a performance boost which was never included in this repo